tests/codebuild/run_all_sample_test.sh

```diff
@@ -65,3 +64,15 @@ if [ "$(kubectl smlogs batchtransformjob xgboost-batch | wc -l)" -lt "1" ]; then
     echo "smlogs batchtransformjob did not produce any output."
     exit 1
 fi
+
+# Clean up resources before re-using metadata names
+delete_all_tests
+
+verify_delete TrainingJob testfiles/xgboost-mnist-trainingjob.yaml
+verify_delete HyperparameterTuningJob testfiles/xgboost-mnist-hpo.yaml
+
+# Create model before running batch delete test
+run_test testfiles/xgboost-model.yaml
+verify_test Model xgboost-model 1m Created
+yq w -i testfiles/xgboost-mnist-batchtransform.yaml "spec.modelName" "$(get_sagemaker_model_from_k8s_model xgboost-model)"
+verify_delete BatchTransformJob testfiles/xgboost-mnist-batchtransform.yaml 60s
```

Comment:
I'm amazed that this only takes 60 seconds / Training and HPO take 30 seconds. Should we put this higher to account for SageMaker p99s?

Reply:
It only barely starts any of the jobs before it tries to delete them. So the only lag is between the controller and SageMaker. It's likely these values will need to be adjusted, though.

Reply:
++ I think setting 60s for all of them would be great, left up to you though